incubator/monochart/values.yaml

```
@@ -67,6 +67,7 @@ deployment:
   #   name: value
   pod:
     # securityContext: {}
+    # hostAliases: {}
```

It is a tiny point, but still, I like documentation to be accurate: `hostAliases` is an array, so it should be `hostAliases: []`. Better:
```suggestion
    # hostAliases:
    #  - ip: "10.1.2.3"
    #    hostnames:
    #    - "foo.remote"
    #    - "bar.remote"
```